lookupapi/defaultsettings.py

```diff
@@ -35,42 +35,42 @@
 
 """
 
-LOOKUP_API_OAUTH2_CLIENT_ID = None
+OAUTH2_CLIENT_ID = None
```

Comment:
The default settings for OAUTH2_{...}, the associated system checks and tests should be moved to automationoauthclient and automationauthdrf as appropriate. The documentation for automationoauthclient still talks about ``ASSETS_OAUTH2_...`` settings.